pkg/provider/provider.go

```diff
@@ -1231,3 +1266,178 @@ func parseLiveInputs(live, oldInputs *unstructured.Unstructured) *unstructured.U
 	metadata.AdoptOldAutonameIfUnnamed(inputs, live)
 	return inputs
 }
+
+// convertPatchToDiff converts the given JSON merge patch to a Pulumi detailed diff.
+func convertPatchToDiff(patch, oldLiveState map[string]interface{}, gvk schema.GroupVersionKind) (map[string]*pulumirpc.PropertyDiff, error) {
```

Comment:
```suggestion
func convertPatchToDiff(patch, oldLiveState map[string]interface{}, gvk schema.GroupVersionKind,
) (map[string]*pulumirpc.PropertyDiff, error) {
```

Reply:
Why add a line break here? This line is still within the 120-character limit, right?

Reply:
It's showing up at 142 for me.

Reply:
Oh, weird. I wonder why the linter isn't catching that.